### PR TITLE
Add language line for the "present" validation rule.

### PR DESCRIPTION
**Pull request by themsaid**
Adding a language line for the newly added "present" validation rule. Refer to https://github.com/laravel/framework/pull/12487

### CODE REVIEW

File: resources/lang/en/validation.php [hunk 1]
```diff
@@ -58,6 +58,7 @@
     'not_in'               => 'The selected :attribute is invalid.',
     'numeric'              => 'The :attribute must be a number.',
     'regex'                => 'The :attribute format is invalid.',
+    'present'              => 'The :attribute field must be present.',
```
Comment (GrahamCampbell) [inline resources/lang/en/validation.php]: Not sure this is the most user friendly message.

Comment (themsaid) [inline resources/lang/en/validation.php]: Any suggestions?

Comment (GrahamCampbell) [inline resources/lang/en/validation.php]: D'no. Taylor is normally good at this sort of thing.

Comment (lagbox) [inline resources/lang/en/validation.php]: Maybe `The :attribute field is not present.`

**Comment by gaomd** [comment]
I think without *field* word should be enough, check the `numeric` line 2 lines above the added line.

**Comment by taylorotwell** [comment]
I think these should also be in alphabetical order.

**Comment by themsaid** [comment]
Sorted. Didn't notice the sorting in the docs and in here, will take care from now on.

**Pull request merged**